packages/test/test-service-load/src/optionsMatrix.ts

```diff
@@ -38,11 +38,12 @@ export function generateRuntimeOptions(seed: number) {
         maxOpsSinceLastSummary: numberCases,
     };
 
-    const runtimeOptionsMatrix: OptionsMatrix<IContainerRuntimeOptions> = {
+    type OptionsUnderTest = Pick<IContainerRuntimeOptions, "gcOptions" | "summaryOptions">;
```

Comment:
i didn't notice this, but this will not pick up new options, one of the points of this is it is strongly typed, and when people add new options they get a compile issus if they do not add them here for the stress test

Reply:
I see -- I'll put up a PR a bit later to flip this from a Pick to an Omit?

Reply:
https://github.com/microsoft/FluidFramework/pull/6141